### PR TITLE
chore: drop support for PHP 7.4

### PR DESCRIPTION
**Pull request by kesselb**
Ref https://github.com/nextcloud/groupware/issues/41

Needs:
- [x] https://github.com/nextcloud/mail/pull/7863
- [x] https://github.com/ChristophWurst/kitinerary-bin/pull/4
- [x] https://github.com/ChristophWurst/kitinerary-sys/pull/2
- [x] https://github.com/ChristophWurst/kitinerary-flatpak/pull/2

### CODE REVIEW

**Comment by ChristophWurst** [comment]
> chore!

Nitpick: 7.4 is EOL so this isn't a breaking change for a Nextcloud app as documented at https://docs.nextcloud.com/server/latest/developer_manual/app_publishing_maintenance/release_process.html#versioning

We will not have to bump the major version number for this change.

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		"christophwurst/kitinerary-flatpak": "^0.3",
 		"christophwurst/kitinerary-sys": "^0.2.0",
 		"ezyang/htmlpurifier": "4.16.0",
-		"gravatarphp/gravatar": "^2.0",
+		"gravatarphp/gravatar": "dev-master#6b9f6a45477ce48285738d9d0c3f0dbf97abe263",
```
[the remark this answers is not in the document]
Comment (kesselb) [reply composer.json:32]: https://github.com/gravatarphp/gravatar/pull/17 is merged but not released.

File: appinfo/info.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
 	<repository type="git">https://github.com/nextcloud/mail.git</repository>
 	<screenshot>https://user-images.githubusercontent.com/1374172/79554966-278e1600-809f-11ea-82ea-7a0d72a2704f.png</screenshot>
 	<dependencies>
-		<php min-version="7.4" max-version="8.1" />
+		<php min-version="8.0" max-version="8.2" />
```
[the remark this answers is not in the document]
Comment (kesselb) [reply appinfo/info.xml:32]: True. Let me move it to another pr.

**Review by ChristophWurst — APPROVED** [review]
Awesome

**Pull request merged**